test-snap-build/action.yaml

```diff
@@ -10,26 +10,55 @@ inputs:
     description: "Option to install the snap on the runner after build"
     default: "false"
     required: false
+  snapcraft-yaml-path:
+    description: "Custom path to snapcraft.yaml allowing to have multiple manifests in one repo."
```

Comment:
```suggestion
    description: Path to snapcraft.yaml, enabling use-cases where it may be in a non-standard location, or where there are multiple snapcraft.yaml files in a given repo.
```